src/api/app/policies/label_template_policy.rb

```diff
@@ -0,0 +1,22 @@
+class LabelTemplatePolicy < ApplicationPolicy
+  class Scope
+    def initialize(user, project)
+      @user = user
+      @project = project
+    end
+
+    def resolve
+      raise Pundit::NotAuthorizedError unless Flipper.enabled?(:labels, @user)
+      raise Pundit::NotAuthorizedError unless @project.maintainers.include?(@user)
+
+      @project.label_templates
+    end
+  end
+
+  def index?
+    raise Pundit::NotAuthorizedError unless Flipper.enabled?(:labels, @user)
```

Comment:
This just raises an error for people that aren't in beta, doesn't it?

Reply:
Hm, true, using it in the tabs makes everythin unusable...

Reply:
Well, actually this works for people on the beta that have the feature flag `labels` enabled.